conf/messages.en

```diff
@@ -1616,16 +1621,16 @@ financialDetailsCheckYourAnswersController.heading = Check your answers
 financialDetailsCheckYourAnswersController.viewOnly.title = Financial details
 financialDetailsCheckYourAnswersController.viewOnly.heading = Financial details
 financialDetailsCheckYourAnswersController.viewOnly.link = View previous version of financial details (opens in new tab)
-financialDetailsCheckYourAnswersController.totalCashInStartDate = Total cash {0} held in all bank accounts on {1}
-financialDetailsCheckYourAnswersController.totalCashInStartDate.hidden = total cash in all bank accounts, on {0}
-financialDetailsCheckYourAnswersController.totalCashInEndDate = Total cash {0} held in all bank accounts on {1}?
-financialDetailsCheckYourAnswersController.totalCashInEndDate.hidden = total cash in all bank accounts, on {0}
-financialDetailsCheckYourAnswersController.valueOfAssetsInStartDate = Value of {0}’s assets, minus liabilities, on {1}?
-financialDetailsCheckYourAnswersController.valueOfAssetsInStartDate.hidden = value of scheme assets, minus liabilities, on {0}
-financialDetailsCheckYourAnswersController.valueOfAssetsInEndDate = What was the total value of {0}’s assets, minus liabilities, on {1}?
-financialDetailsCheckYourAnswersController.valueOfAssetsInEndDate.hidden = total value of scheme assets, minus liabilities, on {0}
-financialDetailsCheckYourAnswersController.feeCommissionWagesSalary = How much did {0} pay in fees or commissions of over 10%?
-financialDetailsCheckYourAnswersController.feeCommissionWagesSalary.hidden = how much the scheme paid in fees or commissions of over 10%
+financialDetailsCheckYourAnswersController.totalCashInStartDate = Total cash the scheme held in all bank accounts on {0}
+financialDetailsCheckYourAnswersController.totalCashInStartDate.hidden = total cash in all bank accounts on {0}
+financialDetailsCheckYourAnswersController.totalCashInEndDate = Total cash the scheme held in all bank accounts on {0}
+financialDetailsCheckYourAnswersController.totalCashInEndDate.hidden = total cash in all bank accounts on {0}
+financialDetailsCheckYourAnswersController.valueOfAssetsInStartDate = Value of the scheme’s assets minus liabilities on {0}
```

Comment:
Should this be "the pension scheme's" ?

Reply:
You're right, thank you for spotting this! I'm a bit confused as to how this happened, given that I copied and pasted over each line yesterday afternoon. Perhaps I used a version of the content doc that was in the process of being edited? In any case, I can amend these in the separate Update #4 branch.